test/unit-tests/function/signal/freqz.test.js

```diff
@@ -0,0 +1,32 @@
+import approx from '../../../../tools/approx.js'
+import assert from 'assert'
+import math from '../../../../src/defaultInstance.js'
+
+const freqz = math.freqz
+
+describe('freqz', function () {
```

Comment:
Can you have a look at the code coverage report? It shows that parts of the function `freqz` are not tested. Can you add tests where needed?

https://app.codecov.io/gh/josdejong/mathjs/pull/2988